src/coffee/directives/api/models/child/marker-child-model.coffee

```diff
@@ -47,6 +47,7 @@ angular.module('uiGmapgoogle-maps.directives.api.models.child')
         else
           action = new PropertyAction (calledKey, newVal) =>
```

Comment:
small bug here where clicked marker was not updating as there was no key, forcing the key all allows updated to propagate through.